lib/main_common.pm

```diff
@@ -577,7 +577,7 @@ sub load_jeos_tests {
         unless (get_var('INSTALL_LTP')) {
             loadtest "console/force_scheduled_tasks";
             loadtest "jeos/grub2_gfxmode";
-            loadtest 'jeos/revive_xen_domain' if check_var('VIRSH_VMM_FAMILY', 'xen');
+            loadtest 'jeos/revive_xen_domain' if (check_var('VIRSH_VMM_FAMILY', 'xen') && (get_var('EXTRATEST') !~ m/filesystem/));
```

Comment:
What is this test actually needed for?

Reply:
Not really sure, but I wouldn't like to decommission this test case in this PR. It is used in most XEN tests. In the filesystems test suite we do not really want to reboot before `snapper_undochange` as the kernel parameters introduced by `bootloader_svirt.pm` would vanish.

Reply:
That it passes without this test in the xen + filesystem case proves that it's not neccessary though, doesn't it?

Reply:
Ok, so let me try also `xen-hvm` + fs test suite without it. :)

Reply:
Seems to be fine http://kepler.suse.cz/tests/1654#

Reply:
Ok, so +1 for removing `revive_xen_domain`